README.md

```diff
@@ -1,55 +1,149 @@
-# COVID-19 CLI
+# CoVID-19 CLI
 
 <img src="https://badges.frapsoft.com/os/v1/open-source.png?v=103"> <a href="LICENSE"><img alt="GitHub" src="https://img.shields.io/github/license/dmahajan980/covid19"></a>
 
 A command-line interface to check the current statistics of COVID-19 of the world or a particular country.
 
-## TABLE OF CONTENTS
-1. [Getting Started](#getting-started)
-2. [Usage](#usage)
-3. [Contributing](#contributing)
-5. [License](#license)
+## Table of Contents
 
-## GETTING STARTED
+1.  [Getting Started](#getting-started)
+2.  [Usage](#usage)
+3.  [Contributing](#contributing")
+4.  [License](#license)
```

Comment:
Same comment about extra whitespace here. Kindly change this at all the places with extra whitespace.

Reply:
This double whitespace is being added by `Prettier`.
Should I manually remove them then?

Reply:
Yes, please. That'd be fine, given that you used Prettier.